mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -183,7 +187,7 @@ def handle_fallback(self, message):
     @staticmethod
     def __find_value(pods, pod_id):
```

Comment:
I think it makes more sense to rename this method to `find_pod_id`, since we're look for ids rather than general values.